File: langs/qr.go
// deps: go get github.com/skip2/go-qrcode
package main

import (
  "fmt"
  "github.com/skip2/go-qrcode"
)

func main() {
  data := "https://example.com"
  err := qrcode.WriteFile(data, qrcode.Medium, 240, "qr.png")
  if err != nil { panic(err) }
  fmt.Println("qr.png")
}

File: langs/qr.rs
// deps: cargo add qrcode image
use qrcode::QrCode;
use image::Luma;

fn main() {
  let data = "https://example.com";
  let code = QrCode::new(data.as_bytes()).unwrap();
  let image = code.render::<Luma<u8>>().build();
  image.save("qr.png").unwrap();
  println!("qr.png");
}
